### PR TITLE
[seo] change doc to best practices

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: bundles/seo/configuration.rst [hunk 1]
```diff
@@ -402,3 +402,5 @@ Whether or not the the :ref:`bundles-seo-alternate-locale` should be loaded
 **type**: ``string`` **default**: ``null``
 
 Specify the service id of a custom :doc:`AlternateLocaleProvider <../seo/alternate_locale>`.
+
+.. _phpcr: ../phpcrodm-admin
```
Comment (wouterj) [inline bundles/seo/configuration.rst]: why did you add this? This should be transformed into a ``:doc:`../phpcrodm-admin` `` link

File: bundles/seo/error_pages.rst [hunk 2]
```diff
@@ -150,22 +150,22 @@ Now, register this new class as a service and tag it as
 
         # app/config/services.yml
         services:
-            app.suggestions.hompage:
+            app.suggestions_hompage:
```
Comment (wouterj) [inline bundles/seo/error_pages.rst]: `homepage_suggestions` ?

File: bundles/seo/introduction.rst [hunk 3]
```diff
@@ -359,24 +359,24 @@ For changing the default translation domain (messages), you should use the
...
-            translation_domain: AcmeDemoBundle
+            translation_domain: AppBundle
 
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
         <container xmlns="http://symfony.com/schema/dic/services">
             <config xmlns="http://cmf.symfony.com/schema/dic/seo"
-                translation-domain="AcmeDemoBundle"
+                translation-domain="AppBundle"
```
Comment (wouterj) [inline bundles/seo/introduction.rst]: Actually, I would use the default `messages` translation domain for AppBundle translations

File: bundles/seo/sitemap.rst [hunk 4]
```diff
@@ -107,7 +107,7 @@ The templates are specified for a specific sitemap (see below) or in the
             sitemap:
                 defaults:
                     templates:
-                        html: AppBundle:Sitemap:default.html.twig
+                        html: ::itemap/default.html.twig
```
Comment (wouterj) [inline bundles/seo/sitemap.rst]: Should be `sitemap/default.html.twig`

File: bundles/seo/sitemap.rst [hunk 5]
```diff
@@ -118,7 +118,7 @@ The templates are specified for a specific sitemap (see below) or in the
             <config xmlns="http://example.org/schema/dic/cmf_seo">
                 <sitemap>
                     <defaults>
-                        <template format="html">AppBundle:Sitemap:default.html.twig</template>
+                        <template format="html">::sitemap/default.html.twig</template>
```
Comment (wouterj) [inline bundles/seo/sitemap.rst]: `::` are not needed

File: bundles/seo/sitemap.rst [hunk 6]
```diff
@@ -127,15 +127,15 @@ The templates are specified for a specific sitemap (see below) or in the
...
-                        'html' => 'AppBundle:Sitemap:default.html.twig',
-                    ),
-                ),
-            ),
-        ));
+        $container->loadFromExtension('cmf_seo', [
+            'sitemap' => [
+                'defaults' => [
+                    'templates' => [
+                        'html' => '::sitemap/default.html.twig',
```
Comment (wouterj) [inline bundles/seo/sitemap.rst]: same here

File: bundles/seo/sitemap.rst [hunk 7]
```diff
@@ -253,7 +253,7 @@ sitemaps, you need to explicitly specify ``sitemap`` as well:
                     categories:
                         default_change_frequency: hourly
                         templates:
-                            html: AppBundle:Sitemap:categories.html.twig
+                            html: ::sitemap/categories.html.twig
```
Comment (wouterj) [inline bundles/seo/sitemap.rst]: and in these 3 lines

**Comment by dbu** [comment]
fixed against 2. 0 branch in #805
